.github/workflows/pr-opened.yml

```diff
@@ -0,0 +1,36 @@
+name: on-pr-opened
+on:
+  pull_request:
+
+jobs:
+  lint-commits:
+    name: lint-commits
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
```

Comment:
I'm not sure we need to enforce this, I think the yarn install below will simply use the nvmrc file. But I might be mistaken ❓